src/components/popover-manager/popover-manager.tsx

```diff
@@ -32,7 +32,7 @@ export class PopoverManager {
    *
    * @default `[data-calcite-popover-reference]`
    */
-  @Prop() selector = "[data-calcite-popover-reference]";
```

Comment:
This component is deprecated so maybe we don't need to update it?

Reply:
I had a note to exclude deprecated props, but it was simple to set up that I just went for it for the greater good. 

![greater-good](https://user-images.githubusercontent.com/197440/188718477-03aab480-0a04-47f3-8d03-b09d80eca113.gif)

Reply:
Sounds good to me 👍